src/Lykke.Service.HFT.Core/AppSettings.cs

```diff
@@ -114,4 +113,17 @@ public class FeesSettings
     {
         public string TargetClientId { get; set; }
     }
+
+    public class AssetsServiceClient
+    {
+        public string ServiceUrl { get; set; }
+    }
+    public class BalancesServiceClient
+    {
+        public string ServiceUrl { get; set; }
+    }
+    public class FeeCalculatorServiceClient
+    {
+        public string ServiceUrl { get; set; }
+    }
```

Comment:
I believe all these classes should be declared in it's own service client packages accordingly.

Reply:
Sure, I'll replace these classes when related services will expose them.